File: streaming/flink/src/main/scala/com/mitosis/beans/FlightInfoBean.scala
package com.mitosis.beans

import scala.beans.{ BeanProperty }

class FlightInfoBean {

  @BeanProperty
  var departingId: String = _

  @BeanProperty
  var arrivingId: String = _

  @BeanProperty
  var tripType: String = _

  @BeanProperty
  var departureDate: String = _

  @BeanProperty
  var arrivalDate: String = _

  @BeanProperty
  var passengerNumber: Int = _

  @BeanProperty
  var cabinClass: String = _
}

File: streaming/spark/src/main/java/com/mitosis/FlightInfoBean.java
package com.mitosis;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
public class FlightInfoBean {

  private Long rowKey;
  private String departingId;
  private String arrivingId;
  private String tripType;
  private Long departureDate;
  private Long arrivalDate;
  private Integer passengerNumber;
  private String cabinClass;
}
